scripts/devops_tasks/tox_harness.py

```diff
@@ -112,7 +112,7 @@ def collect_tox_coverage_files(targeted_packages):
 
     # coverage report has paths starting .tox and azure
     # coverage combine fixes this with the help of tox.ini[coverage:paths]
-    combine_coverage_files(targeted_packages)
+    # combine_coverage_files(targeted_packages) # Skipping this, upload all .coverage-{package-name} files
```

Comment:
I think we need agree on how `coverage` artifacts will be handled. Specifically, in some places, we comment out `combine` of coverage. However, we still talk about as if we were combining down below

```
logging.info("Visible uncombined .coverage files: {}".format(coverage_files))
```

Reply:
Changed this to logging information about what files are found.